python/tskit/tables.py

```diff
@@ -3754,3 +3754,8 @@ def ibd_segments(
             store_pairs=store_pairs,
             store_segments=store_segments,
         )
+
+
+def load_metadata(file):
```

Comment:
I don't think there's much point in providing the ``load_metadata`` function to be honest - it's just as easy to call

```python
metadata = tskit.load(path, skip_tables=True).metadata
```

It just doesn't seem worth having the extra function to document since it's already a one-liner and people might want to access other top-level info from the tree sequence too.